runner/src/python.rs

```diff
@@ -51,11 +50,67 @@ impl AsRef<OsStr> for ColorChoice {
     }
 }
 
+#[cfg(feature = "clap")]
+impl ValueEnum for ColorChoice {
+    fn value_variants<'a>() -> &'a [Self] {
+        &[Self::Auto, Self::Always, Self::Never]
+    }
+
+    fn to_possible_value(&self) -> Option<PossibleValue> {
+        Some(match self {
+            Self::Auto => PossibleValue::new("auto"),
+            Self::Always => PossibleValue::new("always"),
+            Self::Never => PossibleValue::new("never"),
+        })
+    }
+}
+
+#[derive(Copy, Clone, Debug, Default, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum LinkMode {
+    #[default]
+    Copy,
+    Clone,
+    Hardlink,
```

Comment:
Is `copy` the default behavior as of `v0.4.0`? Also, do you think each platform should have their own platform? (copy on windows, hardline on others)

Reply:
It's not clear what the defaults are in the newest version of `uv` from checking the docs, but I would say `copy` is not the most performant, but its the most stable across platforms, so I'd like to use that as a default if it makes sense

Reply:
Totally agree ! Thanks for your explanation